wordpress-mobile/gutenberg-mobile
# Allow shrink mobile toolbar

Fixes #2180 

Ref to gutenberg: https://github.com/WordPress/gutenberg/pull/22132
Ref to WordPress-IOS: https://github.com/wordpress-mobile/WordPress-iOS/pull/14062
Ref to testing ios: https://github.com/wordpress-mobile/WordPress-iOS/pull/14073

NOTE: That PR is adding a patch to RN since currently there is no option for disabling buttons within action sheet ios. PR in RN is waiting to be approved and merge.

To test:

1. Open mobile app
2. Add Buttons block
3. Focus Button block inside Buttons block
4. Expect to see stacked options in the mobile toolbar
5. Press button in the mobile toolbar
6. Expect the disabled button `Move Left` for the first button
7. Expect the disabled button `Move Right` for the last button
8. Try all the options:
	* expect Button is moving left/right when these options are available
	* expect to be able to open settings
	* expect to be able to remove block

PR submission checklist:
